routes/web.php

```diff
@@ -90,9 +90,9 @@
     return view('impact-stories');
 })->name('impact-stories');
 
-Route::get('/newsroom', function () {
+Route::get('/blogs', function () {
```

Comment:
There's already a blog route! Remove this and remove the view for newsroom. I don't think it's needed.